.github/CONTRIBUTING.md

```diff
@@ -59,26 +59,28 @@ The basic workflow for contributing is:
    ```
    git checkout -b name-of-your-branch
    ```
-4. Install the dependencies listed in `requirements.txt` and `requirements-dev.txt`.
+4. Install the [anaconda](https://www.anaconda.com/distribution/#download-section) to avoid any external library errors.
```

Comment:
Let's recommend people to use the `miniconda` distribution instead:

https://docs.conda.io/en/latest/miniconda.html

Reply:
Done.